crates/oxc_linter/src/rules/jsx_a11y/prefer_tag_over_role.rs

```diff
@@ -0,0 +1,137 @@
+use std::collections::HashMap;
+
+use crate::{context::LintContext, rule::Rule, utils::has_jsx_prop_lowercase, AstNode};
+use oxc_ast::{
+    ast::{JSXAttributeItem, JSXAttributeValue, JSXElementName},
+    AstKind,
+};
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint-plugin-jsx-a11y/prefer-tag-over-role: Prefer using semantic HTML tags over `role` attribute.")]
+#[diagnostic(
+    severity(warning),
+    help("Replace HTML elements with `role` attribute to corresponding semantic HTML tags.")
```

Comment:
we probably want to use the `tag`/`role` in here

Reply:
see https://github.com/oxc-project/oxc/blob/128406a8ab6a2b780c317d20a74693141e77933e/crates/oxc_linter/src/rules/unicorn/prefer_string_trim_start_end.rs#L11-L14 for an example

Reply:
Thanks!!!
Fixed in [fix: tag and role](https://github.com/oxc-project/oxc/pull/1831/commits/cefc5fbe2f8e1d2548595e90efee21813e718814)!!